lib/box-node-sdk.js

```diff
@@ -195,10 +195,11 @@ BoxSDKNode.prototype.configure = function(params) {
  * @param {string} accessToken A user's Box API access token
  * @returns {BoxClient} Returns a new Box Client paired to a new BasicAPISession
  */
-BoxSDKNode.prototype.getBasicClient = function(accessToken) {
+BoxSDKNode.getBasicClient = function(accessToken) {
 	var apiSession = new BasicAPISession(accessToken, this.tokenManager);
 	return new BoxClient(apiSession, this.config, this.requestManager);
```

Comment:
I'm not sure this will work as expected — `this.config` and `this.requestManager` are instance properties of the SDK object, and won't be present in a static method like this.

Reply:
Ah yes, the test passed but not sure that tested the right thing. Let me try this again and actually test it properly :)